2025/webextensions-wg.html

```diff
@@ -66,91 +79,125 @@
+              <i class="todo"
+                >[dd monthname yyyy] (date of the "Call for Participation", when
+                the charter is approved)</i
+              >
             </td>
           </tr>
           <tr id="CharterEnd">
-            <th>
-              End date
-            </th>
+            <th>End date</th>
             <td>
               <i class="todo">[dd monthname yyyy]</i> (Start date + 2 years)
             </td>
           </tr>
           <tr>
-            <th>
-              Chairs
-            </th>
+            <th>Chairs</th>
             <td>
-              Timothy Hatcher (Apple)<br>
+              Timothy Hatcher (Apple)<br />
               Simeon Vincent (Mozilla)
             </td>
           </tr>
           <tr>
-            <th>
-              Team Contacts
-            </th>
+            <th>Team Contacts</th>
             <td>
-              <a href="mailto:mike@w3.org">Michael[tm] Smith</a> (0.10 <abbr title="Full-Time Equivalent">FTE</abbr>)
+              <a href="mailto:mike@w3.org">Michael[tm] Smith</a> (0.10
+              <abbr title="Full-Time Equivalent">FTE</abbr>)
             </td>
           </tr>
           <tr>
-            <th>
-              Meeting Schedule
-            </th>
+            <th>Meeting Schedule</th>
             <td>
-              <strong>Teleconferences:</strong> 1-hour calls will be held every two weeks.
-              <br>
-              <strong>Face-to-face:</strong> we will meet during the W3C's annual Technical Plenary week; additional face-to-face meetings may be scheduled by consent of the participants, usually no more than 3 per year.
+              <strong>Teleconferences:</strong> 1-hour calls will be held every
+              two weeks.
+              <br />
+              <strong>Face-to-face:</strong> we will meet during the W3C's
+              annual Technical Plenary week; additional face-to-face meetings
+              may be scheduled by consent of the participants, usually no more
+              than 3 per year.
             </td>
           </tr>
         </table>
-
       </div>
 
       <div id="background" class="background">
         <h2>Motivation and Background</h2>
-        <p>Users can extend their browsers with extensions: small applications that run within a browser that can modify its behaviors and alter page content. Inspired by improvements to interoperability over the last few years, there is a desire to even more closely align these implementations. We aim to address this interest and enable cross-browser extension development through the creation of a specification of a common core of WebExtensions APIs.
+        <p>
+          Users can extend their browsers with extensions: small applications
+          that run within a browser that can modify its behaviors and alter page
+          content. Inspired by improvements to interoperability over the last
```

Comment:
```suggestion
          Users can extend their browsers with extensions: small applications that
          run within a browser that can change the behaviour of the user agent,
          alter the content of web pages and add to and replace features of the browser
          interface. Inspired by improvements to interoperability over the last
```

Reply:
Explicitly mentioning the ability of extensions to alter and add to the browser user interface as not to skip this common oversight (like the way permissions are handled).

Reply:
@carlosjeurissen, I don't quite follow what you're trying to get at with this suggestion. As I currently see it, extensions don't directly alter or add to the browser's user interface. Rather, browsers expose a few places where extensions can display custom content, such as in the sidebar, action icon, or popup. To my mind "the ability … to alter and add to the browser user interface" would more accurately describe IE toolbars and Firefox's XUL-based extension system. 

Also, I'm not sure what you mean by "the way permissions are handled." Do you mean that browsers use a different permission management system for extensions than form websites?

Reply:
I've attempted to address your feedback in https://github.com/w3c/charter-drafts/pull/711/commits/9af31b973ac9c936725f6e05f69322e144fe2547. The first sentence now reads:

> Users can extend their browsers with extensions: small applications that run within a browser that modify its behavior, change web page content, and add or enhance interface features.